src/functionalTest/groovy/com/autonomousapps/jvm/ExplodeJarSpec.groovy

```diff
@@ -0,0 +1,47 @@
+package com.autonomousapps.jvm
+
+import com.autonomousapps.jvm.projects.ExplodeJarProject
+import com.autonomousapps.kit.utils.Files
+
+import java.util.stream.Collectors
+
+import static com.autonomousapps.utils.Runner.build
+import static com.google.common.truth.Truth.assertThat
+
+final class ExplodeJarSpec extends AbstractJvmSpec {
```

Comment:
I'm a little worried about the fragility of this test. Any change to the relevant models will break the test, but that, to me, is just noise—not useful information.

I think you should just delete this spec, the `ExplodeJarProject`, and the two new resources files as well. I don't think they add anything useful when compared to the maintenance cost.